### PR TITLE
BUG 8768094, BUG 8813873 Updated opacity value to card title to fix color contrast issue

### PR DESCRIPTION
**Pull request by v-baambati**
1.Updated opacity value to card title to fix color contrast issue BUG 8768094.

### CODE REVIEW

File: packages/react-charting/src/components/LineChart/eventAnnotation/LabelLink.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const LabelLink: React.FunctionComponent<ILabelLinkProps> = props => {
 
   return (
     <>
-      <g ref={gRef} onClick={onClick} data-is-focusable={false} style={{ cursor: 'pointer' }}>
+      <g ref={gRef} onClick={onClick} data-is-focusable={true} style={{ cursor: 'pointer' }}>
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/LineChart/eventAnnotation/LabelLink.tsx:74]: change file is missing

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by AtishayMsft** [comment]
Need to get the up and down arrows working for entries inside the events callout.

File: packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts [hunk 2]
```diff
@@ -128,7 +128,7 @@ export const getStyles = (props: ICartesianChartStyleProps): ICartesianChartStyl
       {
         ...fonts.small,
         lineHeight: '16px',
-        opacity: '0.8',
+        opacity: '0.85',
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts:131]: change file update is missing

File: packages/react-charting/src/components/GaugeChart/GaugeChart.styles.ts [hunk 3]
```diff
@@ -76,7 +76,7 @@ export const getStyles = (props: IGaugeChartStyleProps): IGaugeChartStyles => {
       {
         ...theme.fonts.small,
         lineHeight: '16px',
-        opacity: '0.8',
+        opacity: '0.85',
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/GaugeChart/GaugeChart.styles.ts:79]: test snapshots also need to be updated for this change

Comment (v-baambati) [reply packages/react-charting/src/components/GaugeChart/GaugeChart.styles.ts:79]: updated

File: change/@fluentui-react-charting-d3660b0d-d2af-403a-bc19-a05978feeee9.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-d3660b0d-d2af-403a-bc19-a05978feeee9.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31307&commitId=bf954d7ae7e31432583d5c454431dce9712fc6a9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (AtishayMsft) [reply change/@fluentui-react-charting-d3660b0d-d2af-403a-bc19-a05978feeee9.json:1]: 2 new unrelated screenshots added.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**